streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -626,13 +628,19 @@ public void shouldPunctuateActiveTasks() {
     }
 
     @Test
-    public void shouldResumeConsumptionOfInitializedPartitions() {
+    public void shouldNotResumeConsumptionUntilAllStoresRestored() {
```

Comment:
For this test would it be more clear to define a local strict mock for the consumer `Consumer<byte[], byte[]> consumer = EasyMock.createStrictMock(Consumer.class);` then get rid of lines 637-643?

Reply:
Updated the code to use strict mock.